tests/test_serializer.py

```diff
@@ -31,7 +32,8 @@ def test_invalid_serializer(self):
         serializer = self.Serializer(data={'char': 'abc'})
         assert not serializer.is_valid()
         assert serializer.validated_data == {}
-        assert serializer.errors == {'integer': ['This field is required.']}
+        assert serializer.errors['integer'].detail == ['This field is required.']
```

Comment:
Well what I mean is that I could keep the new `ValidationError` in `_errors` and change the definition of `errors` so that it will keep returning the same type as before and avoid backward incompatibilities.